pkg/virt-api/validating-webhook/validating-webhook.go

```diff
@@ -41,6 +42,7 @@ const (
 	cloudInitMaxLen = 2048
 	arrayLenMax     = 256
 	maxNetworks     = 1
+	maxStrLeng      = 36
```

Comment:
maybe rename s/maxStrLeng/maxStrLen/g

also, the limit is more of a sanity check than anything else. I think the limit should be high enough that someone would never hit it unless they're doing something strange.

I'd be in favor of a 256 character limit.

Reply:
Sounds good to me